packages/react/src/components/Authenticator/SetupTOTP/SetupTOTP.tsx

```diff
@@ -25,7 +25,7 @@ export const SetupTOTP = ({
   className,
   variation,
 }: RouteProps): JSX.Element => {
-  const { fields, getTotpSecretCode, isPending, user } = useAuthenticator(
+  const { getTotpSecretCode, isPending, user, QRFields } = useAuthenticator(
```

Comment:
Is this a new external facing API, or is this internal only?

Reply:
This is internal only.